src/libraries/System.Numerics.Tensors/tests/TensorPrimitives.Generic.cs

```diff
@@ -392,15 +388,21 @@ public static IEnumerable<object[]> SpanDestinationFunctionsToTest()
             yield return new object[] { new SpanDestinationDelegate(TensorPrimitives.Log10P1), new Func<T, T>(T.Log10P1) };
             yield return new object[] { new SpanDestinationDelegate(TensorPrimitives.RadiansToDegrees), new Func<T, T>(T.RadiansToDegrees) };
             yield return new object[] { new SpanDestinationDelegate(TensorPrimitives.Reciprocal), new Func<T, T>(f => T.One / f) };
-            yield return new object[] { new SpanDestinationDelegate(TensorPrimitives.ReciprocalEstimate), new Func<T, T>(T.ReciprocalEstimate), T.CreateTruncating(1.171875e-02) };
+            // TODO https://github.com/dotnet/runtime/issues/98861
+            yield return new object[] { new SpanDestinationDelegate(TensorPrimitives.ReciprocalEstimate), new Func<T, T>(T.ReciprocalEstimate), Helpers.DetermineTolerance<T>(doubleTolerance: 1e-3, floatTolerance: 1e-3f) };
```

Comment:
Some of the tested methods have very low precision, which is why I'm marking every test case with a TODO.

Reply:
The `*Estimate` functions are notably expected to be low precision. They explicitly trade accuracy for speed and are exposed so users who want the perf can get it.

Reply:
What would be a good baseline for that class of methods?

Reply:
Ideally we have similar tolerances as to what we use in the official tests.

I would expect that for `*Estimate` functions, we can exactly match. I would expect for other functions we may need to be slightly more flexible, but ideally we stay as close to the original as possible and note which APIs are causing the needed difference.

Reply:
Presumably you mean these?

https://github.com/dotnet/runtime/blob/89f7ad3b276fb0b48f20cb4e8408bdce85c2b415/src/libraries/System.Runtime/tests/System.Runtime.Extensions.Tests/System/Math.cs#L14-L45

Reply:
Right. In particular I expect `CrossPlatformMachineEpsilonForEstimates` can be used exactly for the `*Estimate` functions.

I expect `CrossPlatformMachineEpsilonSingle` and `CrossPlatformMachineEpsilonDouble` can be mostly used as is, but that we might have a little bit more variance than that with some of the current naive implementations.

Reply:
Ok, I've pushed updates adopting the epsilons used in the scalar test suite which didn't impact the number of diverging implementations substantially. Currently, the following vectorized routines require lower tolerances:

| Routine | Double $\epsilon$ |  Single $\epsilon$ |
| ---------|----------------------|---------------------|
| `Cbrt` |  `1e-13` | N/A |
| `Cos` | `1e-4` | N/A |
| `Cosh` | `1e-14` | N/A |
| `CosPi` | `1e-4` | `1e-4` |
| `Exp10` | `1e-6` | `1e-5` |
| `Exp2` | `1e-6` | `1e-5` |
| `ExpM1` | `1e-6` | N/A |
| `Exp2M1` | `1e-6` | `1e-5` |
| `Exp10M1` | `1e-6` | `1e-5` |
| `Pow` (scalar/span) | `1e-4` | N/A |
| `Pow` (span/span) | `1e-13` | `1e-4` |
| `Pow` (span/scalar) | `1e-13` | `1e-3` |
| `RootN` | `1e-13` | N/A |
| `Sin` | `1e-13` | N/A |
| `Sinh` | `1e-14` | N/A |
| `SinPi` | `1e-13` | `1e-4` |
| `Tan` | `1e-13` | `1e-4` |